map_file: /home/ubuntu/catkin_ws/src/ui_package/maps/Group1/Map3.yaml
route_file: /home/ubuntu/catkin_ws/src/ui_package/paths/Group1/Map3/RouteTest.csv
